docker-compose-web-modeler.yaml

```diff
@@ -125,11 +125,9 @@ services:
       CLIENT_PUSHER_FORCE_TLS: "false"
       CLIENT_PUSHER_KEY: modeler-app-key
       OAUTH2_CLIENT_ID: web-modeler
+      OAUTH2_JWKS_URL: http://keycloak:8080/auth/realms/camunda-platform/protocol/openid-connect/certs
       OAUTH2_TOKEN_AUDIENCE: web-modeler
       OAUTH2_TOKEN_ISSUER: http://localhost:18080/auth/realms/camunda-platform
-      KEYCLOAK_BASE_URL: http://localhost:18080
-      KEYCLOAK_REALM: camunda-platform
```

Comment:
is KEYCLOAK_REALM not necessary anymore?

Reply:
@jessesimpson36: We're using a new library for the authentication now that doesn't require the realm to be configured separately any more. It's already part of `OAUTH2_TOKEN_ISSUER` and `OAUTH2_JWKS_URL` anyway.